src/components/game/container.ts

```diff
@@ -0,0 +1,100 @@
+import styled from "styled-components";
+import left from "../../../resources/images/left-bg.png";
+
+const GameContainer = styled.div`
+  display: grid;
+  grid-template-columns: 1fr 1fr;
+  font-family: "Roboto Mono", monospace;
+  .right {
+    display: none;
+  }
+  .left {
+    display: block;
+    width: 207px;
+    margin: 0em 1px 25em 0em;
+    background-image: url(${left});
+    background-size: cover;
+    background-position: center;
+    background-repeat: no-repeat;
+    overflow-x: hidden;
+    overflow-y: scroll;
+    .container {
+        display:none;
+        padding: 1em 1em;
+    }
+  }
+  .middle {
+    color: white;
+    text-align: center;
+    button {
+      background: #1c1d22;
+      border: none;
+    }
+    p {
+      text-align: left;
+      font-size: 12px;
+      padding-left: 2.8em;
+    }
+    .cookieText {
+      color: #858080;
+      text-align: left;
+      margin-top: -10px;
+      padding: 0 0 3em 0;
+      font-size: 9px;
+      padding-left: 3.4em;
+    }
+    .cookieBtn {
+      height: 25vmin;
+      pointer-events: none;
+      animation: animate 7s linear infinite;
+      background: #1c1d22;
```

Comment:
Please remove this background because it is not well displayed when resizing:
<img width="661" alt="image" src="https://user-images.githubusercontent.com/45454460/199449360-1b78a3b7-1aeb-43d2-a2b9-13ce11f88108.png">